src/eval.jl

```diff
@@ -1,6 +1,7 @@
 using CodeTools, LNR, Media
 import CodeTools: getthing
 
+preprocess(x) = x
```

Comment:
Yes, that would still work. It would have the added benefit though that you could check if the preprocessor has been changed by checking `Atom.preprocess == identity`. If it's a unique function `preprocess(x) = x`, you don't have a reference to check against.